machine-learning-box/NBA_metrics/queries/cart_abandon.sql

```diff
@@ -0,0 +1,46 @@
+SELECT
+      DISTINCT td_client_id,
+      4 AS code
+    FROM (
+        SELECT
+          td_client_id,
+          session_id,
+          row_number(
+          ) over(
+            partition BY session_id,
+            td_client_id
+          ORDER BY
+            time DESC
+          ) AS rnum,
+          flag
+        FROM (
+            SELECT
+              td_client_id,
+              TD_SESSIONIZE_WINDOW(
+                time,
+                3600 * 24
+              ) OVER(
+                PARTITION BY td_client_id,
+                td_ip
+              ORDER BY
+                time
+              ) AS session_id,
+              time,
+              (
+                CASE
+                  WHEN td_url = 'https://www.varidesk.com/cart' THEN 1
```

Comment:
This should be a variable like ${cart_url}

Reply:
@toru-takahashi you should not be seeing any files from the NBA_metrics box at all. This is an older folder with old version of queries that I deleted from my branch on my computer. I am not sure why it still comes up during the pull request... :(   I need some help figuring out why you're seeing this folder.